taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -0,0 +1,22 @@
+import useFetch from "../../components/useFetch";
+import { API_URL } from "../../constants";
+import EntriesList from "./EntriesList";
+
+const Entry = () => {
+    const { data: nodes, isPending, isError, isSuccess, errorMessage } = useFetch(API_URL+'nodes');
+    if (isError) {
+        return (
+            <div className="all-entries">
+                {isError && <div>{errorMessage}</div>}
+            </div>
+        )
+    }
+    return (
+        <div className="all-entries">
+            {isPending && <div>Loading...</div>}
+            {isSuccess && <EntriesList nodes={nodes} title={"Test"}/>}

```

Comment:
nit: if there's no plan to include more things in this component, I think the content of `EntriesList` could live here (as is `EntriesList` is mainly `jsx` it should't be much of a problem)